app/views/layouts/application.html.erb

```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
     <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag    'application', media: 'all',
+                                              'data-turbolinks-track': 'reload' %>
```

Comment:
1. trong file rails dùng nháy đôi nhé
file js, css thì nháy đơn

2. indent line 7 chỉ tab vào 1 tab so với lien 6 thôi

3. Các thành phần phân tách nhau 1 space thôi -> Sau `stylesheet_link_tag` là 1 space

4. Tìm hiểu I18n và sử dụng I18n CHO TOÀN BỘ TEXT CỦA HTML file nhé